### DISCUSSION TITLE
关于bpmn插件

### DISCUSSION THREAD

**Discussion by jrdamocles** [didi/LogicFlow#1315] [General]
最近有需求要在实现审批流，找来找去感觉logicflow比较合适，扩展性比较好，看了下文档，好像有BpmnElements插件和BpmnAdapter插件，以及新版本的BpmnElements和BpmnAdapter，但是文档里写的还是比较模糊，看起来使用新的插件会比较好？不知道有没有更详细一点的使用和设计文档呢

**Comment by wumail** [comment]
暂时只有文档里的那些内容，目前看大家普遍还在用旧的插件，后续会更新的详细一点，欢迎使用新的bpmn插件，使用过程中有问题都可以及时反馈~

**Reply by jrdamocles** [reply]
好的，感谢回复🙏，初步使用了一下，是满足目前的需求的

**Comment by jrdamocles** [comment]
https://juejin.cn/post/7275550548944764965 发了一篇新BPMN插件的文章，大家可以参考一下

**Reply by wumail** [reply]
牛蛙牛蛙
